pkgs/development/node-packages/default.nix

```diff
@@ -273,7 +273,7 @@ let
         wrapProgram "$out/bin/prisma" \
           --prefix PRISMA_MIGRATION_ENGINE_BINARY : "${prisma-engines}/bin/migration-engine" \
```

Comment:
Also `--set-default` for all maybe https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/make-wrapper.sh ?

Reply:
Or `--set` I personally don't care.